helper/helper.py

```diff
@@ -3,11 +3,27 @@
 from sqlalchemy import MetaData, create_engine
 from sqlalchemy.orm import sessionmaker
 
-DB_URL = 'postgresql://root:root@db:5432/perfectfit'
+DB_URL_DEFAULT = 'postgresql://root:root@db:5432/perfectfit'
 
+def santize_db_url(db_url):
+    """
+    SQLAlchemy cannot handle the "postgres" prefix
+    (see https://github.com/sqlalchemy/sqlalchemy/issues/6083), but Heroku still populates
+    DATABASE_URL with that protocol designator. For such URLs we need to modify the string
+    from "postgres" -> "postgresql".
 
-def get_db_session(db_url=DB_URL):
-    engine = create_engine(db_url)
+    We are doing this string replacement because the config var DATABASE_URL provided by
+    Heroku may change.
+    """
+    if db_url.startswith("postgres://"):
+        db_url = db_url.replace("postgres://", "postgresql://", 1)
+
+    return db_url
```

Comment:
Could you make use of this function in `dbschema/migrations/env.py`?

Reply:
We were indeed considering that, but the problem is that this gives an 'out of package' import which isn't really safe/clean either, so for now it's probably best to just duplicate it (these will be the only 2 instances) with appropriate comment.

Reply:
Check